pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TableConfig.java

```diff
@@ -101,6 +102,9 @@ public class TableConfig extends BaseJsonConfig {
   @JsonPropertyDescription(value = "Configs for Table config tuner")
   private List<TunerConfig> _tunerConfigList;
 
+  @JsonPropertyDescription(value = "Name of the table-group")
+  private String _tableGroupName;
```

Comment:
Did we consider referring to TableGroupConfig directly here ? If null, then current behavior. If non null, then table is part of some TableGroup. See the example of `FieldConfig` which by itself is like a sub-config but is referred to from TableConfig.

Reply:
Yeah that was in fact the first implementation I had. I ended up changing this since multiple tables can be part of a table-group and that'd lead to duplication of the group-config.